specification/specs/x_pack/transform/get_transform/GetTransformRequest.ts

```diff
@@ -25,6 +25,7 @@
 interface GetTransformRequest extends RequestBase {
   path_parts?: {
     transform_id?: Name
+    exclude_generated?: boolean
```

Comment:
This should go in the [querystring](https://www.elastic.co/guide/en/elasticsearch/reference/current/get-transform.html#get-transform-query-parms).

Reply:
Yes, sorry about that and great catch!